src/app/modules/gb-export-module/gb-export.component.ts

```diff
@@ -95,7 +93,17 @@ export class GbExportComponent implements OnInit, OnDestroy {
     return this.exportService.exportJobs;
   }
 
-  get isLoadingExportDataTypes(): boolean {
-    return this.exportService.isLoadingExportDataTypes;
+  get isDataTypesUpdating(): boolean {
+    return this.exportService.isDataTypesUpdating;
+  }
+
+  get isDataTableUpdating(): boolean {
+    return this.exportService.isDataTableUpdating;
+  }
+
+  get isExportCreationUIShown(): boolean {
+    const isDataTableExportAllowed = !(this.isExternalExportAvailable || this.isTransmartSurveyTable);
```

Comment:
maybe easier to just replace it with:
```suggestion
    const isDataTableExportAllowed = this.isTransmartDataTable;
```
?
Then there will be no need to update this part each time we add a new export mode/type

Reply:
good catch, I meant to change this part but forgot :D